### ISSUE TITLE
App startup | LOA1 Gate

### ISSUE THREAD

**Issue by bessgreen** [department-of-veterans-affairs/va.gov-team#15100]
## User story
As a user, I want to know what documents/information I'll need to provide to VA in order to access the app. 

Dev note: this is just an informational page. The user will verify their identity through id.me. 

## Acceptance Criteria
1. Selecting "Sign in" from the Unauthorized Home page routes to a new page called Sign In
2. Styling & UI matches the design
     - Copy on this page is hard-coded
3. "Back" button routes back to the Unauthorized Home page
4. Display Veteran Crisis Line banner
5. Accordion: "Read more if you haven't yet verified"
     - Collapsed, on default
     - When expanded, this contains hard-coded copy and a bulleted list
6. "Continue to sign in" large button (UI only - routing handled in a separate story)
7. Rules for when this page appears in the auth flow:
     - This page is triggered from clicking "Sign in" on the app's unauthenticated home screen (this page: https://adhoc.invisionapp.com/share/YUYRT9BAPSB#/screens/432863239)
     - This LOA1 page will only display in the auth flow the **first time** a user is signing into the app
     - If a first-time user is on the LOA1 page, hits the Back button, and then hits "Sign in" from the unauthenticated home screen, they should still see the LOA1 page. Only after the user has signed into the app should this LOA1 page be hidden from that user's future login flows

## Designs
LOA1 page: https://adhoc.invisionapp.com/share/YUYRT9BAPSB#/screens/433563910
Unauthorized home page that precedes this page: https://adhoc.invisionapp.com/share/YUYRT9BAPSB#/screens/432863239

## Accessibility Acceptance Criteria
1. Everything on the page should be read by the screen reader.
     - If anything is being conveyed visually only (buttons, menu items, list, tables, images, headings, forms, etc) the component type needs to be announced by the screenreader. (Include name (title), role, value)
2. Each page should support text resizing based on the OS font scaling settings without loss of content, content overlap, or horizontal scroll.
3. Each touch target must be at minimum 44px by 44px
4. The app needs to support operability by tap (no gestures) or voice control only. In cases where gestures occur, e.g. scroll, ensure these simple gestures are operable through the OS’ built-in capabilities.  
5. All elements on a page should be able to be traversed by an auxiliary keyboard. As a user uses the keyboard to navigate a page, its focus element should be clear.
6. When a page opens, the first section or element should automatically be placed in focus
7. When a page opens, the title page should be announced

## Accessibility Implementation Guidelines 

- Mark "Sign In" as Header 1

**Back button**
- accessibilityLabel= "Back"
- accessibilityHint= " Go back to the unauthorized home page"
- accessibilityRole= "button" 

**Veterans Crisis Line**
- Title = “Talk to the Veterans Crisis Line now”
- accessibilityHint = “Go to Veterans Crisis Line”
- accessibilityRole = “button”

**Read more if you haven't yet verified component**
- accessibilityLabel="read more if you haven't yet verified"
- accessibilityHint="view what you will need to verify your identity"
- accessibilityRole="spinbutton"
- accessibilityState= "collapsed"
- Denote when dropdown is expanded
- Ensure dropdown area does not "trap" the keyboard focus

**Continue to sign in button**
- accessibilityLabel= “continue to sign in”
- accessibilityHint= "navigates to the sign in page"
- accessibilityRole= “button”

## Dev checklist
- [ ] UX/UI
- [ ] Accessibility
- [ ] Add tests

**Comment by bessgreen** [comment]
hi @bridgethapner, are you able to review/validate AC #7?

**Comment by bridgethapner** [comment]
@bessgreen AC 7s is correct!

**Comment by digitalken** [comment]
PASS - UI on Android (small bug logged on text bolding) and Accessibility.
Still waiting on iOS build to have this in it.

Minor bug logged:
https://app.zenhub.com/workspaces/va-mobile-app-5f205eac3155a2000f422ecb/issues/department-of-veterans-affairs/va.gov-team/18902
